zpay32/testutil/testutil.go

```diff
@@ -0,0 +1,134 @@
+package testutil
```

Comment:
I think this should have a name that ends with `_test.go` to avoid cluttering the normal API.

Reply:
Also, I don't think you need a new folder just for this file.